### PR TITLE
CIV-4209 | add new resolve dispute screen and related urls and tests

### PR DESCRIPTION
**Pull request by lfazelyniaHMCTS**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-4209

### Change description ###
New screen to be added which informs the user on how to try and resolve the dispute

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/modules/i18n/locales/cy.json [hunk 1]
```diff
@@ -421,6 +421,24 @@
       "REJECT_WHOLE_CLAIM": "Os ydych yn gwrthod yr hawliad cyfan, gall yr hawliad gael ei drosglwyddo i ganolfan wrandawiadau Llys Sirol.",
       "RESPONSE_MEANING": "Canfod beth mae pob ymateb yn ei olygu"
     },
+    "CLAIM_JOURNEY": {
```
Comment (hmcts-version1-domini) [inline src/main/modules/i18n/locales/cy.json:424]: The CLAIM_JOURNEY section should go under the CLAIM_DETAILS section due to alphabetical order.

File: src/main/modules/i18n/locales/en.json [hunk 2]
```diff
@@ -421,6 +422,24 @@
       "REJECT_WHOLE_CLAIM": "If you reject all of the claim, the claim may be transferred to a County Court hearing centre.",
       "RESPONSE_MEANING": "Find out what each response means"
     },
+    "CLAIM_JOURNEY": {
```
Comment (hmcts-version1-domini) [inline src/main/modules/i18n/locales/en.json:425]: Same here. CLAIM_JOURNEY go under the CLAIM_DETAILS

File: src/main/views/features/claim/resolving-this-dispute.njk [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          <h2 class="govuk-heading-m">{{ t('PAGES.CLAIM_JOURNEY.RESOLVE_THE_DISPUTE.P2.HEADING') }}</h2>
+          <p class="govuk-body">{{ t('PAGES.CLAIM_JOURNEY.RESOLVE_THE_DISPUTE.P2.TRY_TO_RESOLVE') }}</p>
+          <ul class="govuk-list govuk-list--bullet">
+            <li>{{ t('PAGES.CLAIM_JOURNEY.RESOLVE_THE_DISPUTE.P2.TELLING_THEM_WHY') }}</li>
+            <li>{{ t('PAGES.CLAIM_JOURNEY.RESOLVE_THE_DISPUTE.P2.SUGGESTING_TIMETABLE') }}</li>
+            <li>{{ t('PAGES.CLAIM_JOURNEY.RESOLVE_THE_DISPUTE.P2.MAKE_A_CLAIM') }}</li>
+          </ul>
+          {{ govukButton({
+            text: t("COMMON.BUTTONS.READ_CONFIRMATION"),
+            href: urlNextView
```
Comment (commitBlob) [inline src/main/views/features/claim/resolving-this-dispute.njk:32]: where is `urlNextView` coming from?

File: src/test/unit/routes/features/claim/resolvingThisDisputeController.test.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import nock from 'nock';
+import config from 'config';
+import {
+  CLAIM_RESOLVING_DISPUTE_URL,
+} from '../../../../../main/routes/urls';
+import {TestMessages} from '../../../../utils/errorMessageTestConstants';
+import {mockCivilClaim} from '../../../../utils/mockDraftStore';
+
+jest.mock('../../../../../main/modules/oidc');
+jest.mock('../../../../../main/modules/draft-store');
```
Comment (commitBlob) [inline src/test/unit/routes/features/claim/resolvingThisDisputeController.test.ts:12]: if you are not using claim details in your view or controller you don't need `draft-store` mock

File: src/test/unit/views/features/claim/resolving-this-dispute.test.ts [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+import {CLAIM_RESOLVING_DISPUTE_URL} from '../../../../../main/routes/urls';
+import {mockCivilClaim} from '../../../../utils/mockDraftStore';
+import {t} from 'i18next';
+
+const jsdom = require('jsdom');
+const {JSDOM} = jsdom;
+const pageTitle = 'PAGES.CLAIM_JOURNEY.RESOLVE_THE_DISPUTE.PAGE_TITLE';
+
+jest.mock('../../../../../main/modules/oidc');
+jest.mock('../../../../../main/modules/draft-store');
```
Comment (commitBlob) [inline src/test/unit/views/features/claim/resolving-this-dispute.test.ts:14]: if you are not using claim details in your view or controller you don't need `draft-store` mock

File: src/test/utils/errorMessageTestConstants.ts [hunk 6]
```diff
@@ -53,6 +53,7 @@ export const enum TestMessages {
   FIRST_PAYMENT_DATE_IN_THE_FUTURE_REQUIRED = 'Enter a first payment date in the future',
   PAGE_NOT_FOUND = 'Page Not Found',
   PAYMENT_FREQUENCY_REQUIRED = 'Choose a payment frequency',
+  RESOLVING_DISPUTE = 'Try to resolve the dispute',
```
Comment (commitBlob) [inline src/test/utils/errorMessageTestConstants.ts:56]: is this error message?

File: src/main/views/features/claim/resolving-this-dispute.njk (hunk 3 above)
Comment (hmcts-version1-domini) [reply src/main/views/features/claim/resolving-this-dispute.njk:32]: AC of the task says that the button shouldn't do anything for now. I guess this way isn't doing anything, but in my case, for example, I just put text only and the button does nothing already.

Comment (commitBlob) [reply src/main/views/features/claim/resolving-this-dispute.njk:32]: Ok, are we keeping track of this anywhere?

File: src/test/utils/errorMessageTestConstants.ts (hunk 6 above)
Comment (lfazelyniaHMCTS) [reply src/test/utils/errorMessageTestConstants.ts:56]: No, I've removed it and the part of the test associated with it

File: src/main/views/features/claim/resolving-this-dispute.njk (hunk 3 above)
Comment (lfazelyniaHMCTS) [reply src/main/views/features/claim/resolving-this-dispute.njk:32]: I've removed this anyway as it is redundant code

**Review by RosDiaf — APPROVED** [review]

File: src/test/unit/views/features/claim/resolving-this-dispute.test.ts (hunk 5 above)
Comment (lfazelyniaHMCTS) [reply src/test/unit/views/features/claim/resolving-this-dispute.test.ts]: Removed

File: src/test/unit/routes/features/claim/resolvingThisDisputeController.test.ts (hunk 4 above)
Comment (lfazelyniaHMCTS) [reply src/test/unit/routes/features/claim/resolvingThisDisputeController.test.ts]: Removed

File: src/main/modules/i18n/locales/cy.json (hunk 1 above)
Comment (lfazelyniaHMCTS) [reply src/main/modules/i18n/locales/cy.json]: I've moved it now

File: src/main/modules/i18n/locales/en.json [hunk 7]
```diff
@@ -36,7 +36,8 @@
       "RESPOND_TO_CLAIM": "Respond to claim",
       "SAVE_AND_CONTINUE": "Save and continue",
       "START_AGAIN": "Start again",
-      "SUBMIT_RESPONSE": "Submit Response"
+      "SUBMIT_RESPONSE": "Submit Response",
+      "READ_CONFIRMATION": "I confirm I've read this"
```
Comment (commitBlob) [inline src/main/modules/i18n/locales/en.json:40]: you are missing this key in the welsh json

Comment (lfazelyniaHMCTS) [reply src/main/modules/i18n/locales/en.json:40]: Added, and made sure it's in alphabetical order as well

**Review by GorilaNaranja — APPROVED** [review]

**Review by commitBlob — APPROVED** [review]

**Pull request merged**